client/app/components/conference_room/provider/EventProvider.jsx

```diff
@@ -90,6 +91,7 @@ export default class EventProvider extends React.Component {
     return (
       <Component currentEvent={this.state.currentEvent}
                  nextEvents={this.state.nextEvents}
+                 allEvents={this.state.allEvents}
```

Comment:
Well, allEvents are events from all the conference rooms, so the first option is not possible. We could actually do it the other way but a lot of logic in EventProvider is based on state.currentEvent so I guess it's not worth doing it